intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -128,19 +129,29 @@ We might say...
 
 ### State and Behavior
 
-When we start defining our own classes, we will specify their **state** and their generic **behavior.**
+When we start defining our own classes, we will specify two main things: their **state** and their **behavior**.
+
+In the context of Object-Oriented Programming, **state** is used to refer to all the variables packaged within a class and **behavior** is used to refer to all the functions packaged within a class.
```

Comment:
This is getting a little implementation-specific-y, but given that we need to talk about Attributes and Methods since the vocabulary has been brought up, this little spoiler section feels ok. (It just took me a while of thinking about it to realize that's what I thought!)